pkgs/by-name/hi/hifiscan/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+let
+  inherit (python3Packages) numpy sounddevice pyqt6 pyqt6-sip pyqtgraph;
+
+  eventkit = let
```

Comment:
Is there a reason this shouldn't be a top-level package in `pkgs/development/python-modules`?

Reply:
Kind of isn't, however it probably won't be used by anything apart from that utility. 

I am not even sure that the package itself should be in by-path, since it's clearly something which should be `callPackage`d by pythonPackages. Guess I wanted to try by-path :)

Reply:
I think it makes sense to leave the `hifiscan` program in `by-name`, but the python module might be used by other programs so it should at least be exposed in the global package set somehow. At the moment, `by-name` doesn't include python libraries, only applications, so maybe you could move `eventkit` to a new file at `pkgs/development/python-modules/eventkit/default.nix`?

Reply:
Yeah, I did some more research, and turned out that eventkit might actually get used by someone else. So I guess I'll just move everything to pythonPackages in the end

Reply:
Moved it)